src/Dice.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Random;
+
+/**
+ * Created by melmo on 11/29/16.
+ */
+public class Dice {
+    private int sides;
+
+    public Dice(int sides){
+        this.sides = sides;
+    }
+
+    public int roll() {
+        Random rand = new Random();
```

Comment:
As discussed in class this will create a new `Random` object every single time the roll is called.  This would better be served as a class attribute that the constructor simply instantiates.